test/js-native-api/test_object/test.js

```diff
@@ -296,6 +296,9 @@ assert.deepStrictEqual(test_object.TestGetProperty(), {
   obj.x = 'd';
 }
 
+if (common.isFreeBSD())
+  common.skip('The following is failing on FreeBSD for now obvious reason.');
```

Comment:
```suggestion
  common.skip('The following is failing on FreeBSD for no obvious reason.');
```

Reply:
Temporary commit.. not worth reviewing yet... not done testing